src/olympia/reviewers/templatetags/code_manager_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag
+def code_manager_url(
+        page, addon_id, version_id, compare_version_id=None, file=None):
+    # Always return URLs in en-US because the Code Manager is not localized.
+    url = '{}/en-US'.format(settings.CODE_MANAGER_URL)
```

Comment:
Can't we have locale-free URLs?

Reply:
It looks like we cannot. If I try to go to a code manager page without the `/en-US/` I get an "Ooops, page not found!" page. That's not ideal, but also not something I think is worth working on at this point. TBH I'm not sure why we dealt with locales at all for code manager. I guess the theory was that at some point it could possibly be localizable.